tests/platforms/linux/system/test_run.py

```diff
@@ -359,7 +359,7 @@ def test_run_console_app(run_command, first_app, tmp_path):
     run_command.run_app(first_app, test_mode=False, passthrough=[])
 
     # The process was started
-    run_command.tools.subprocess.run.mock_calls == [
+    assert run_command.tools.subprocess.run.mock_calls == [
```

Comment:
Wow... that's a pretty glaring error...

Reply:
The best kinds of tests are the ones you can't fail! :laughing: